include/nfc/tnep/tag.rst

```diff
@@ -35,34 +35,47 @@ The client (application or upper-layer protocol) must define these services and
 At least one TNEP service must be defined for the initial NDEF message to be created.
 
 To create a TNEP service, use the :c:macro:`NFC_TNEP_TAG_SERVICE_DEF` macro.
+The service data is stored in the persistent storage.
 
-After defining the services, create a table that lists all of them.
-You can use :c:macro:`NFC_TNEP_TAG_SERVICE` to get the names of the created services.
+Initial TNEP NDEF message
+=========================
 
-The service table is given to the :cpp:func:`nfc_tnep_tag_initial_msg_create` function and the Service Parameter record of each service is inserted into the initial TNEP message.
+The services data are loaded when calling the :cpp:func:`nfc_tnep_tag_initial_msg_create` function and the Service Parameter record of each service is inserted into the initial TNEP message.
 A Service Parameter record contains parameters for communicating with the service.
+The additional NDEF Records can be also encoded into the Initial TNEP message, as follows:
 
-See the following code for an example of service definition and TNEP initialization:
+* Pass a callback to :cpp:func:`nfc_tnep_tag_initial_msg_create`.
+* Next, define an additional Record inside a callback and call :cpp:func:`nfc_tnep_initial_msg_encode`
+
+See the following code for an example of the Initial Message creation:
```

Comment:
What is the difference here?
See the following code for an example of the Initial Message creation (this line)
vs 
The following code shows how to create the Initial NDEF Message (line 57)

Reply:
Not fixed yet, some additional clarification are needed. Anyway, the idea was to not have 2 different blocks of code one after the other without any text description between them 
@KAGA164 FYA

Reply:
The first sentence shall say encoding not creation. I fixed it